SEED-platform/seed
# Standardize inventory detail display name

#### Any background context you want to provide?

#### What's this PR do?
- Creates a reusable html component for all Inventory Detail pages.

There are essentially only 2 changes being made in this PR, however they are made in several places
- html files: include the new reusable header component
- controllers: consistent variable assignment for variables used in the above component

#### How should this be manually tested?
Go to inventory detail, navigate through all tabs with various default display fields, the headers should be consistent

#### What are the relevant tickets?
#4710 

#### Screenshots (if appropriate)
Old:
<img width="576" alt="Screenshot 2024-08-23 at 11 18 10 AM" src="https://github.com/user-attachments/assets/fba32260-1e1b-4481-a028-77ddbb0b0e57">


New: Unknown 
<img width="606" alt="Screenshot 2024-08-23 at 11 17 28 AM" src="https://github.com/user-attachments/assets/e7558ae2-077c-44da-8835-018daa2da42f">

New: Known
<img width="591" alt="Screenshot 2024-08-23 at 11 17 40 AM" src="https://github.com/user-attachments/assets/c2c9e6b2-d2c8-49e6-8103-0f0d9fe497e2">
